### DISCUSSION TITLE
About CHI-cache-transitions.sm

### DISCUSSION THREAD

**Discussion by 1fdmesb** [gem5/gem5#884] [Q&A]
When I read the file "CHI-cache-transitions.sm", I saw these codes:
transition({UD_RSC,SD_RSC,UC_RSC,UD_RU,UC_RU,UD_RSD}, Global_Eviction, BUSY_BLKD) {ReplTBEAvailable} {
  ......
}
transition(SC_RSC, Global_Eviction, BUSY_BLKD) {ReplTBEAvailable} {
  ......
}
Well, I found that the SD_RSD state can not meet the Global_Eviction event. But why? Could anybody help me?

**Comment by powerjg** [comment]
I *think* you're describing an error you're seeing or a potential bug. Can you give us more information? Specifically, what are the inputs to gem5, what is the configuration, how did you run gem5, what is the output you see, what is the output you expected?

**Reply by 1fdmesb** [reply]
Well, I am just reading the relevant code  to gather information about the CHI protocol that gem5 has implemented, and I haven't actually run gem5 or configured it. My question is as stated above, that is, I noticed in the CHI-cache-transitions.sm file, there is no code description for SD_RSD state triggering Event: Global_Eviction  . I don't know why this is the case
